README.md

```diff
@@ -248,19 +250,19 @@ rules:
 
 # Service Binding
 
-A Service Binding describes the connection between a [Provisioned Service](#provisioned-service) and a [workload projection](#workload-projection).  It **MUST** be codified as a concrete resource type with API version `servicebinding.io/v1beta1` and kind `ServiceBinding`.  Multiple `ServiceBinding`s can refer to the same service.  Multiple `ServiceBinding`s can refer to the same workload.  For portability, the schema **MUST** comply to the exemplar CRD found [here][sb-crd].
+A Service Binding describes the connection between a [Provisioned Service](#provisioned-service) and a [Workload Projection](#workload-projection).  It **MUST** be codified as a concrete resource type with API version `servicebinding.io/v1beta1` and kind `ServiceBinding`. An implementation **MAY** support other version of this specification which define additional API versions.  Multiple `ServiceBinding`s can refer to the same service.  Multiple `ServiceBinding`s can refer to the same workload.  For portability, the schema **MUST** comply to the exemplar CRD found [here][sb-crd].
 
-Restricting service binding to resources within the same namespace is strongly **RECOMMENDED**.  Implementations that choose to support cross-namespace service binding **SHOULD** provide a security model that prevents attacks like privilege escalation and secret enumeration, as well as a deterministic way to declare target namespaces.
+Restricting service binding to resources within the same namespace is strongly **RECOMMENDED**.  Implementations that choose to support cross-namespace service binding **MUST** provide a security model that prevents attacks like privilege escalation and secret enumeration, as well as a deterministic way to declare target namespaces.
```

Comment:
I definitely agree that all implementations need to have this posture. 👍 

My only concern is with the thought that we can codify each **MUST** as an assertion in a conformance test.  How would we do that here - meaning, how would we assert that an implementation truly prevents this?  That can always be another attack / vulnerability uncovered.

Reply:
It's not something we can enforce because it's inherently implementation specific behavior. This is why we originally had it as a **SHOULD**. Even though we can't enforce it, I still lean on CYA.

Reply:
I think we can work on an extension (post GA) to help cross-namespace support. I just created an issue based on an earlier attempt to create an operator to sync Provisioned Service from one namespace to another: https://github.com/servicebinding/spec/issues/213